src/NUnitFramework/framework/Internal/PlatformHelper.cs

```diff
@@ -214,16 +214,15 @@ public bool IsPlatformSupported( string platform )
                 case "MACOSX":
                     isSupported = _os.IsMacOSX;
                     break;
+
                 // These bitness tests relate to the process, not the OS.
-                // We can't use Environment.Is64BitProcess because it's
-                // only supported in NET 4.0 and higher.
```

Comment:
Not specific to 4.6.2 per se, but something that might've been missed as part of removing .NET 4.0 support